CHANGELOG.md

```diff
@@ -3,6 +3,7 @@
 ## [Unreleased]
 
 -   KFP SDK version bumped to 1.8.11 in order to fix misbehaving TTL issue
+-   Drop support for VertexAI
```

Comment:
add note that it was moved to a separate project

Reply:
Good point!